Source/Data/BPMFMappings.txt

```diff
@@ -132741,7 +132736,7 @@
 音樂系 ㄧㄣ ㄩㄝˋ ㄒㄧˋ
 音樂總監 ㄧㄣ ㄩㄝˋ ㄗㄨㄥˇ ㄐㄧㄢ
 音樂老師 ㄧㄣ ㄩㄝˋ ㄌㄠˇ ㄕ
-音樂課 ㄧㄣ ㄌㄜˋ ㄎㄜˋ
+音樂課 ㄧㄣ ㄩㄝˋ ㄎㄜˋ
```

Comment:
Conflict: https://github.com/openvanilla/McBopomofo/pull/505

Reply:
removed them to avoid conflicts